From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@ClarityQueen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

cjgronlund commented:
@ClarityQueen 
Diff looks weird. Agreed that I'd close without accepting.

Commit:
Adding a missing full stop.

Commit:
Merge pull request #15 from unifycloud/unifycloud-patch-6

Spelling correction

khdang committed:
Update release notes for Java SDK 1.12.0

steved0x committed:
Redis cache - memory considerations

steved0x committed:
Redis cache - memory considerations

steved0x committed:
Redis cache - memory considerations

steved0x committed:
Redis cache - memory considerations

khdang committed:
Updated Java SDK 1.12.0 release notes

steved0x committed:
Redis cache - memory considerations

steved0x committed:
Redis cache - memory considerations

steved0x committed:
Redis cache - memory considerations

steved0x committed:
Redis cache - memory considerations

paulathurman committed:
edit pass: delete-vault

paulathurman committed:
edit pass: delete-vault

Deland-Han committed:
init

Deland-Han committed:
Merge branch 'master' into CI58858

rwike77 committed:
added encrypt text section

rwike77 committed:
updated app manifest

rwike77 committed:
added more config sections

rwike77 committed:
fixed link

Commit:
Add files via upload

Commit:
Create azure-database-security-overview.md

Commit:
Delete azure-database-fig1.png.png

Commit:
Delete azure-database-fig2.png.png

Commit:
Delete azure-database-fig3.png.png

Commit:
Delete azure-database-fig4.png.png

Commit:
Delete azure-database-fig5.jpg.jpg

Commit:
Update azure-database-security-overview.md

Commit:
Add files via upload

Commit:
Update azure-database-security-overview.md

Commit:
Update azure-database-security-overview.md

Commit:
updates on heading 5 .

Commit:
Added metadata.

Commit:
Added metadata.

Commit:
Update azure-database-security-overview.md

rwike77 committed:
update

Commit:
Update log-analytics-usage.md

Added additional audit policies that result in lots of Security Events

Commit:
Update log-analytics-usage.md

trying to fix quote in a table

rwike77 committed:
adding resource governance

Commit:
Added Next steps.

Commit:
Update azure-database-security-overview.md

Commit:
Update azure-database-security-overview.md

rwike77 committed:
resource governance updates

pvrk committed:
Updating doc for general availability

pvrk committed:
Add files via upload

pvrk committed:
Files for File restore GA

pvrk committed:
Update backup-azure-restore-files-from-vm.md

pvrk committed:
Add files via upload

TerryLanfear committed:
TOC update for PR #17656

pvrk committed:
Add files via upload

TerryLanfear committed:
fix

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into unify-cloud

Commit:
Changed H1, metadata

Commit:
More tweaks

cjgronlund committed:
update

cjgronlund committed:
edit

cjgronlund committed:
edits

Commit:
Update log-analytics-usage.md

Fixed code formatted in tables and added more queries to help investigate

cychua committed:
Add release notes for 561

Commit:
Update log-analytics-usage.md

Unable to get the pipe sign within a code block in a table -- pulling the code out to be prior to the table.

Commit:
Update log-analytics-usage.md

removing escaped pipe

Commit:
quickfix02

Commit:
Update log-analytics-usage.md

additional clean up

Commit:
updating relnotes

Commit:
updating relnotes

cychua committed:
Update based on feedback from vsolinux

Commit:
quickfix04

Commit:
Merge pull request #17806 from MarkusVi/quickfix02

quickfix02

Commit:
Merge pull request #17716 from richrundmsft/patch-4

Update log-analytics-usage.md

Commit:
updating relnotes

Commit:
quickfix05

Commit:
Merge pull request #17810 from MarkusVi/quickfix04

quickfix04

Commit:
Merge pull request #17811 from MarkusVi/quickfix05

quickfix05

Commit:
Merge pull request #17809 from HeathL17/hl-relnotes

updating relnotes

Commit:
quickfix06

Commit:
Merge pull request #17813 from MarkusVi/quickfix06

quickfix06

Commit:
quickfix07

Commit:
Merge pull request #17815 from MarkusVi/quickfix07

quickfix07

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

tfitzmac committed:
changed casing

Commit:
Changed the section related to dynamic group rule changes. Updated it to reflect the behavior of incremental membership changes and removed the unnecessary advice to create separate groups.

Commit:
Merge pull request #17057 from khdang/patch-1

Update release notes for Java SDK 1.12.0

Commit:
Merge pull request #17819 from piotrci/GBLImprovements

GBL and dynamic groups improvements

curtand committed:
2 articles - fix links, update metadata

Juliako committed:
Update media-services-media-encoder-premium-workflow-multiplefilesinput.md

cychua committed:
Add limitation on selected attribute to merge objects

GitHubber17 committed:
Merge pull request #17782 from JennieHubbard/patch-72

Changed H1, metadata

GitHubber17 committed:
Merge pull request #17818 from tfitzmac/0720casing

changed casing

Commit:
Document blob indexer's text parsing mode and other improvements

Deland-Han committed:
finalversion

spelluru committed:
updated linked service to use LINUX HDI cluster

spelluru committed:
formatting issues

curtand committed:
fix portal references and 1 procuDedure

curtand committed:
fix portal references and 1 procedure

Commit:
Improved the warning text

curtand committed:
Added Azure Information Protection to Security Administrator role

Commit:
Merge pull request #17829 from piotrci/GBLFixes

Improved the warning text

GitHubber17 committed:
Merge pull request #17821 from curtand/admin0720

2 articles - fix links, update metadata

Commit:
Merge pull request #17830 from curtand/roles0720

Added Azure Information Protection to Security Administrator role

Commit:
Merge pull request #17822 from Juliako/patch-8

Update media-services-media-encoder-premium-workflow-multiplefilesinp…

pvrk committed:
Update backup-azure-restore-files-from-vm.md

Commit:
Merge pull request #17825 from chaosrealm/master

Document blob indexer's text parsing mode and other improvements

Deland-Han committed:
update

Juliako committed:
Update media-services-check-job-progress.md

Juliako committed:
Update media-services-dotnet-create-contentkey.md

Commit:
Merge pull request #17831 from Juliako/patch-8

Update media-services-check-job-progress.md

Commit:
Merge pull request #17832 from Juliako/patch-9

Update media-services-dotnet-create-contentkey.md

Juliako committed:
Update media-services-index-content.md

Juliako committed:
Update media-services-licenses-partner-integration.md

Juliako committed:
Update media-services-mes-preset-H264-Multiple-Bitrate-1080p-Audio-5.1.md

Juliako committed:
Update media-services-mes-preset-H264-Multiple-Bitrate-1080p.md

Commit:
Merge pull request #17835 from Juliako/patch-12

Update media-services-index-content.md

Juliako committed:
Update media-services-mes-preset-H264-Multiple-Bitrate-16x9-SD-Audio-5.1.md

Commit:
Merge pull request #17836 from Juliako/patch-13

Update media-services-licenses-partner-integration.md

Juliako committed:
Update media-services-mes-preset-H264-Multiple-Bitrate-16x9-SD.md

Juliako committed:
Update media-services-mes-preset-H264-Multiple-Bitrate-16x9-for-iOS.md

Commit:
Merge pull request #17837 from Juliako/patch-8

Update media-services-mes-preset-H264-Multiple-Bitrate-1080p-Audio-5.…

Juliako committed:
Update media-services-mes-preset-H264-Multiple-Bitrate-4K-Audio-5.1.md

Juliako committed:
Update media-services-mes-preset-H264-Multiple-Bitrate-4K.md

Commit:
Merge pull request #17838 from Juliako/patch-9

Update media-services-mes-preset-H264-Multiple-Bitrate-1080p.md

Juliako committed:
Update media-services-mes-preset-H264-Multiple-Bitrate-4x3-SD-Audio-5.1.md

Commit:
Merge pull request #17839 from Juliako/patch-14

Update media-services-mes-preset-H264-Multiple-Bitrate-16x9-SD-Audio-…

Juliako committed:
Update media-services-mes-preset-H264-Multiple-Bitrate-4x3-SD.md

Commit:
Merge pull request #17840 from Juliako/patch-12

Update media-services-mes-preset-H264-Multiple-Bitrate-16x9-SD.md

Juliako committed:
Update media-services-mes-preset-H264-Multiple-Bitrate-4x3-for-iOS.md

Commit:
Merge pull request #17841 from Juliako/patch-15

Update media-services-mes-preset-H264-Multiple-Bitrate-16x9-for-iOS.md

Juliako committed:
Update media-services-mes-preset-H264-Single-Bitrate-Low-Quality-SD-for-Android.md

Commit:
Merge pull request #17842 from Juliako/patch-13

Update media-services-mes-preset-H264-Multiple-Bitrate-4K-Audio-5.1.md

Commit:
Merge pull request #17843 from Juliako/patch-16

Update media-services-mes-preset-H264-Multiple-Bitrate-4K.md

Juliako committed:
Update media-services-mes-preset-H264-Single-Bitrate-High-Quality-SD-for-Android.md

Commit:
Merge pull request #17844 from Juliako/patch-8

Update media-services-mes-preset-H264-Multiple-Bitrate-4x3-SD-Audio-5…

Juliako committed:
Update media-services-mes-preset-H264-Single-Bitrate-720p-for-Android.md

Commit:
Merge pull request #17845 from Juliako/patch-9

Update media-services-mes-preset-H264-Multiple-Bitrate-4x3-SD.md

Juliako committed:
Update media-services-mes-preset-H264-Single-Bitrate-720p.md

Commit:
Merge pull request #17846 from Juliako/patch-14

Update media-services-mes-preset-H264-Multiple-Bitrate-4x3-for-iOS.md

Juliako committed:
Update media-services-mes-preset-H264-Single-Bitrate-4x3-SD.md

Commit:
Merge pull request #17847 from Juliako/patch-12

Update media-services-mes-preset-H264-Single-Bitrate-Low-Quality-SD-f…

Juliako committed:
Update media-services-mes-preset-H264-Single-Bitrate-4x3-SD-Audio-5.1.md

Juliako committed:
Update media-services-mes-preset-H264-Single-Bitrate-4x3-SD-Audio-5.1.md

Commit:
Merge pull request #17848 from Juliako/patch-15

Update media-services-mes-preset-H264-Single-Bitrate-High-Quality-SD-…

Juliako committed:
Update media-services-mes-preset-H264-Single-Bitrate-4K.md

Commit:
Merge pull request #17849 from Juliako/patch-13

Update media-services-mes-preset-H264-Single-Bitrate-720p-for-Android.md

Juliako committed:
Update media-services-mes-preset-H264-Single-Bitrate-16x9-SD.md

Commit:
Merge pull request #17850 from Juliako/patch-8

Update media-services-mes-preset-H264-Single-Bitrate-720p.md

Juliako committed:
Update media-services-mes-preset-H264-Single-Bitrate-16x9-SD-Audio-5.1.md

Juliako committed:
Update media-services-mes-preset-H264-Single-Bitrate-1080p.md

Commit:
Merge pull request #17851 from Juliako/patch-16

Update media-services-mes-preset-H264-Single-Bitrate-4x3-SD.md

Juliako committed:
Update media-services-mes-preset-H264-Multiple-Bitrate-720p.md

Commit:
Merge pull request #17852 from Juliako/patch-14

Update media-services-mes-preset-H264-Single-Bitrate-4x3-SD-Audio-5.1.md

Juliako committed:
Update media-services-mes-preset-H264-Multiple-Bitrate-720p-Audio-5.1.md

Commit:
Merge pull request #17853 from Juliako/patch-17

Update media-services-mes-preset-H264-Single-Bitrate-4x3-SD-Audio-5.1.md

Commit:
Merge pull request #17854 from Juliako/patch-12

Update media-services-mes-preset-H264-Single-Bitrate-4K.md

Commit:
Merge pull request #17855 from Juliako/patch-15

Update media-services-mes-preset-H264-Single-Bitrate-16x9-SD.md

Commit:
Merge pull request #17856 from Juliako/patch-13

Update media-services-mes-preset-H264-Single-Bitrate-16x9-SD-Audio-5.…

Commit:
Merge pull request #17857 from Juliako/patch-18

Update media-services-mes-preset-H264-Single-Bitrate-1080p.md

Commit:
Merge pull request #17858 from Juliako/patch-8

Update media-services-mes-preset-H264-Multiple-Bitrate-720p.md

Commit:
Merge pull request #17859 from Juliako/patch-16

Update media-services-mes-preset-H264-Multiple-Bitrate-720p-Audio-5.1.md

anoobbacker committed:
Adding support package URLs

anoobbacker committed:
Update storsimple-8000-system-requirements.md

anoobbacker committed:
Update storsimple-8000-system-requirements.md

anoobbacker committed:
Update storsimple-8000-system-requirements.md

anoobbacker committed:
Update storsimple-8000-system-requirements.md

Commit:
Update azure-database-security-overview.md

anoobbacker committed:
Update storsimple-8000-system-requirements.md

Commit:
Merge pull request #17862 from anoobbacker/patch-5

Adding support package URLs

Commit:
Merge pull request #17793 from cychua/master

Add release notes for 561

anoobbacker committed:
Removing updates alert

anoobbacker committed:
Update storsimple-manage-alerts.md

Commit:
Merge pull request #17828 from spelluru/adfhdiissue

updated linked service to use LINUX HDI cluster

Commit:
Merge pull request #17865 from anoobbacker/patch-7

Update storsimple-manage-alerts.md

Commit:
Merge pull request #17864 from anoobbacker/patch-6

Removing updates alert

Commit:
Update TOC.md

cephalin committed:
usability feedback

cephalin committed:
#17130

cephalin committed:
usability feedback

cephalin committed:
#17036

cephalin committed:
#12746

MattGLaBelle committed:
Merge pull request #17751 from pvrk/patch-11

Updating doc for general availability

v-thepet committed:
Merge pull request #17253 from steved0x/sd713

Redis cache - memory considerations

v-thepet committed:
Merge pull request #17340 from Deland-Han/CI58858

CI 58858

v-thepet committed:
Merge pull request #17656 from unifycloud/azure-database-overview

Azure-database-overview-07192017

MattGLaBelle committed:
Merge pull request #17867 from cephalin/usability

usability feedback

cjgronlund committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into hdi-intro-fix

MattGLaBelle committed:
Merge pull request #17866 from saveenr/patch-4

Update TOC.md

ShawnJackson committed:
Update delete-vault.md

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

iainfoulds committed:
Updating example VM create command per user feedback

v-alje committed:
Merge pull request #17869 from MicrosoftDocs/FromPublicRepo

From public repo

ShawnJackson committed:
Merge pull request #17313 from paulathurman/delete-vault

edit pass: delete-vault

cjgronlund committed:
edits

cjgronlund committed:
edit

rwike77 committed:
updated prereqs and party cluster info

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into unify-cloud

vhorne committed:
add languages

Commit:
Merge pull request #17875 from rwike77/quickstart

updated prereqs and party cluster info

Lisaco88 committed:
Merge pull request #17625 from rwike77/containerupdates

Containerupdates

Lisaco88 committed:
Merge pull request #17770 from TerryLanfear/unify-cloud

TOC update for PR #17656

vhorne committed:
tweak

Lisaco88 committed:
Merge pull request #17788 from cjgronlund/hdi-intro-fix

HDI intro update

ramankumarlive committed:
updated content for snapshots

GitHubber17 committed:
Merge pull request #17870 from iainfoulds/vmlinuxstorage

VM Linux storage overview

Juliako committed:
Update media-services-mes-preset-H264-Single-Bitrate-1080p-Audio-5.1.md

Juliako committed:
Update media-services-mes-preset-H264-Single-Bitrate-4K-Audio-5.1.md

ramankumarlive committed:
updated

Juliako committed:
Update media-services-mes-preset-H264-Single-Bitrate-720p-Audio-5.1.md

cynthn committed:
Update create-vm-generalized-managed.md

Juliako committed:
Update media-services-dotnet-dynamic-manifest.md

Juliako committed:
Update media-services-media-encoder-standard-formats.md

Lisaco88 committed:
Merge pull request #17877 from vhorne/upd-stordev

add languages

Commit:
Merge pull request #17881 from ramankumarlive/patch-1

updated content for snapshots

Juliako committed:
Update media-services-release-notes.md

Commit:
Merge pull request #17887 from cynthn/patch-41

Update create-vm-generalized-managed.md

Commit:
Merge pull request #17882 from Juliako/patch-8

Update media-services-mes-preset-H264-Single-Bitrate-1080p-Audio-5.1.md

Commit:
Merge pull request #17885 from Juliako/patch-13

Update media-services-mes-preset-H264-Single-Bitrate-720p-Audio-5.1.md

Commit:
Merge pull request #17884 from Juliako/patch-12

Update media-services-mes-preset-H264-Single-Bitrate-4K-Audio-5.1.md

Commit:
Merge pull request #17889 from Juliako/patch-15

Update media-services-media-encoder-standard-formats.md

Commit:
Merge pull request #17888 from Juliako/patch-14

Update media-services-dotnet-dynamic-manifest.md

Commit:
Merge pull request #17892 from Juliako/patch-16

Update media-services-release-notes.md
